liquidation-monitor/src/agent.spec.js

```diff
@@ -161,125 +231,400 @@ describe('check agent configuration file', () => {
 
 describe('mock axios POST request', () => {
   it('should call axios.post and return a response', async () => {
-    axios.post.mockResolvedValue('test');
     const response = await axios.post('https://...');
     expect(axios.post).toHaveBeenCalledTimes(1);
-    expect(response).toEqual('test');
+    expect(response.data.pagination_summary.total_entries).toEqual(1);
 
     // reset call count for next test
     axios.post.mockClear();
     expect(axios.post).toHaveBeenCalledTimes(0);
   });
 });
 
+// Mock helper functions
+function setDefaultMocks() {
+  // Set all default mocks
+
+  // Compound API
+  axios.post.mockResolvedValue(mockCompoundResponse);
+
+  const {
+    getAssetsIn,
+    markets,
+    getAccountLiquidity,
+    getRateToEth,
+    decimals,
+    getAccountSnapshot,
+    symbol,
+    underlying,
+    exchangeRateStored,
+  } = mockContract;
+
+  // Comptroller
+  /* eslint-disable new-cap */
```

Comment:
It doesn't look like this comment disabling an ESLint warning needs to be here, as there is no constructor being called.
```suggestion
```